pom.xml

```diff
@@ -18,14 +18,15 @@
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
 
-        <jenkins.version>2.303.3</jenkins.version>
-        <jenkins-tools-bom.artifactId>bom-2.303.x</jenkins-tools-bom.artifactId>
-        <jenkins-tools-bom.version>987.v4ade2e49fe70</jenkins-tools-bom.version>
+        <jenkins.version>2.319.3</jenkins.version>
```

Comment:
We should rely on more recent core already, but probably pending https://github.com/jenkinsci/jobcacher-plugin/issues/76